browser/BUILD.gn

```diff
@@ -66,6 +66,8 @@ source_set("browser_process") {
     "component_updater/brave_component_updater_configurator.h",
     "component_updater/brave_component_updater_delegate.cc",
     "component_updater/brave_component_updater_delegate.h",
+    "farbling/ua_farbling_tab_helper.cc",
```

Comment:
nit: you might want to just call this farbling_tab_helper in case we have more uses for it in the future